protocol/src/main/resources/protocol.xml

```diff
@@ -113,6 +117,14 @@
     <data name="bpmnProcessId" id="1" type="varDataEncoding"/>
   </sbe:message>
 
+  <sbe:message name="AdminRequest" id="40">
+    <field name="partitionId" id="1" type="uint16"/>
+    <field name="type" id="4" type="AdminRequestType"/>
```

Comment:
:question: Any reason id is 4, but not 2?

Reply:
Huh, no, no reason. Probably just a leftover from trying a few different variations.